### ISSUE TITLE
Parsing and writing `INDEPENDENT` block is broken

### ISSUE THREAD

**Issue by Helveg** [BlueBrain/nmodl#1123]
```python
import nmodl

print(nmodl.to_nmodl(nmodl.NmodlDriver().parse_string("INDEPENDENT {t FROM 0 TO 1 WITH 1 (ms)}")))
```

outputs an invalid `INDEPENDENT` block:

```
INDEPENDENT {
t}
```

which `nrnivmodl` gives a syntax error about:

```
syntax error:
 name FROM number TO number WITH integer
 at line 23 in file Cav2_1__0.mod
t}
 ^
Translating Ca__granule_cell.mod into /home/robin/.cache/Glia/92db3105/x86_64/Ca__granule_cell.c
make: *** [makemod2c_inc:35: Cav2_1__0.c] Error 1
make: *** Waiting for unfinished jobs....
Translating Cav2_2__0.mod into /home/robin/.cache/Glia/92db3105/x86_64/Cav2_2__0.c
Thread Safe
Thread Safe
```

**Comment by Helveg** [comment]
@pramodk this even looks like something I could fix, I probably just have to adapt a visitor somewhere and make it output the `FROM` and `TO` nodes. If you point me to the relevant code I can attempt a fix!

**Comment by alkino** [comment]
I point to this commit. IIRC INDEPENDENT is now useless: https://github.com/BlueBrain/nmodl/commit/16a09e185d545f06350999a732ab1349ecdd3a8e

**Comment by ramcdougal** [comment]
Even if `INDEPENDENT` has no meaningful semantics in NEURON 9, I would argue that generating code that does not have semantics ande is not syntactically correct in older versions where the semantics could technically matter is not a great solution.

**Comment by pramodk** [comment]
I "fixed" it in #1124 i.e. 

```console
$ cat test.py

import nmodl
print(nmodl.to_nmodl(nmodl.NmodlDriver().parse_string("NEURON {} INDEPENDENT {t FROM 0 TO 1 WITH 1 (ms)}")))
```

and it will give

```console
NEURON {
}

: INDEPENDENT block is deprecated and has no effect in the NEURON model. Skipped!
```

See my comment in PR for the reasoning. Happy to change it people think we really need to reproduce INDEPENDENT block.
